sentry/src/test/java/io/sentry/hints/AttachmentsTest.kt

```diff
@@ -0,0 +1,60 @@
+package io.sentry.hints
+
+import io.sentry.Attachment
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class AttachmentsTest {
```

Comment:
`m`: Shouldn't these tests rather be in `HintsTest`?

Reply:
Moved